src/i18n/locales/en/translation.json

```diff
@@ -398,8 +398,9 @@
           "private_key": {
             "title": "Manage your key",
             "subtitle": "Paste your private key or generate a new one. Make sure you backup your private key before you generate a new one. <0>Learn more »</0>",
-            "warning": "This will delete your old private key. Are you sure?",
+            "warning": "Please type your account's name to delete your old private key",
             "success": "Private key encrypted & saved successfully.",
+            "failed": "Failed to delete your old private key",
```

Comment:
can you rename this to `failed_to_remove` or `failed_to_delete` as this can be interpreted as failure in key generation

Reply:
Done.